backend/config/settings/base.py

```diff
@@ -22,7 +22,8 @@
 ANALYSIS_DIR = Path(PROJECT_ROOT, 'backend', 'app', 'analysis')
 ANALYSIS_PICKLE_PATH = Path(BACKEND_DIR, ANALYSIS_DIR, 'analysis_results')
 LOCAL_PHOTOS_DIR = "/static/images/photos"
-LOCAL_SRCS_DIR = Path(PROJECT_ROOT, 'assets', 'images', 'photos')
+# LOCAL_SRCS_DIR = Path(PROJECT_ROOT, 'assets', 'images', 'photos')
+LOCAL_SRCS_DIR = Path(PROJECT_ROOT, 'backend', 'data', 'test_photos')
```

Comment:
You have changed `LOCAL_SRCS_DIR` to point to test photos directory and added this as the default value to non-test functions. Is this correct?

Reply:
Looking at `TEST_PHOTOS_DIR` value, I think this is a mistake.

Reply:
yes!

Reply:
as in, I did intentionally change LOCAL_SRCS_DIR to have the same value as TEST_PHOTOS_DIR. should I just say LOCAL_SRCS_DIR = TEST_PHOTOS_DIR instead? (after having defined TEST_PHOTOS_DIR of course)

Reply:
hmm this doesn't seem right. What is the original use of LOCAL_SRCS_DIR? We don't want regular functions (like analysis functions) to default to test directories, we want them to look at the real local photos

Reply:
makes sense, I've changed it back so LOCAL_PHOTOS_DIR (the same as LOCAL_SRCS_DIR) is differentiated from TEST_PHOTOS_DIR and the latter is passed into the test cases as needed